templates/study.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="static/study.css">
+</head>
+<style>
+.btn {
+    border: none;
+    background-color:inherit;
+    font-size: 16px;
+}
+    
+.btn:hover {background:gainsboro;}
+</style>
+
+
+<body>
+    <h1> <!--Title from that specific study set from database--> </h1>
+    <h3>Term</h3>
+    <input type="text" name="Term" value="default value">
+
+    <button onclick="answerFunction()">Answer</button>
+    <p id="answer"></p>
+
+
+
+    <script>
```

Comment:
@Kathrin20 Can you extract this to a separate JavaScript file called study.js under the static folder?